stencil.config.ts

```diff
@@ -34,7 +34,10 @@ export const config: Config = {
     { type: "docs-readme" },
     {
       type: "www",
-      copy: [{ src: "../demos", dest: "demos" }],
+      copy: [
+        { src: "../demos", dest: "demos" },
+        { src: "../../node_modules/@esri/calcite-components/dist", dest: "node_modules/@esri/calcite-components/dist" }
```

Comment:
nitpick: could simplify the dest directory structure to just "calcite" or something - no need to keep the 4 level directory structure.

Reply:
I went with this (simple and gives context)

```js
{ src: "../../node_modules/@esri/calcite-components/dist/calcite", dest: "vendor/@esri/calcite-components" }
```